docker/Dockerfile-workers

```diff
@@ -58,24 +51,24 @@ RUN dnf -y install \
     python3-devel \
     python3-pip \
     skopeo \
+    podman \
+    && dnf update -y \
     && dnf clean all
 
-COPY --from=builder /tmp/podman-1.9.3-2.el8.x86_64.rpm /tmp/podman-1.9.3-2.el8.x86_64.rpm
-RUN dnf -y install /tmp/podman-1.9.3-2.el8.x86_64.rpm
-
 COPY --from=builder /usr/bin/opm /usr/bin/opm
 ADD https://github.com/estesp/manifest-tool/releases/download/v1.0.0/manifest-tool-linux-amd64 /usr/bin/manifest-tool
-ADD https://github.com/fullstorydev/grpcurl/releases/download/v1.7.0/grpcurl_1.7.0_linux_x86_64.tar.gz /src/grpcurl_1.7.0_linux_x86_64.tar.gz
-RUN cd /usr/bin && tar -xf /src/grpcurl_1.7.0_linux_x86_64.tar.gz grpcurl && rm -f /src/grpcurl_1.7.0_linux_x86_64.tar.gz
+ADD https://github.com/fullstorydev/grpcurl/releases/download/v1.8.2/grpcurl_1.8.2_linux_x86_64.tar.gz /src/grpcurl_1.8.2_linux_x86_64.tar.gz
+RUN cd /usr/bin && tar -xf /src/grpcurl_1.8.2_linux_x86_64.tar.gz grpcurl && rm -f /src/grpcurl_1.8.2_linux_x86_64.tar.gz
 RUN chmod +x /usr/bin/manifest-tool
-ADD https://github.com/operator-framework/operator-sdk/releases/download/v1.9.0/operator-sdk_linux_amd64 /usr/bin/operator-sdk
+ADD https://github.com/operator-framework/operator-sdk/releases/download/v1.10.1/operator-sdk_linux_amd64 /usr/bin/operator-sdk
 RUN chmod +x /usr/bin/operator-sdk
 
 # Adjust storage.conf to enable Fuse storage.
 RUN sed -i -e 's|^#mount_program|mount_program|g' /etc/containers/storage.conf
 COPY docker/libpod.conf /usr/share/containers/libpod.conf
 
 COPY . .
+RUN pip3 install --upgrade pip
```

Comment:
I believe we are already installing pip with dnf. It will install the latest one. This can be removed, no?

Reply:
this is pip upgrade. It should stay

Reply:
Sounds good. Could you please add a comment here.

Reply:
added the comment